tests/phpunit/tests/bookmark/getBookmark.php

```diff
@@ -0,0 +1,384 @@
+<?php
+
+/**
+ * @group bookmark
+ * @covers ::get_bookmark
+ */
+class Tests_Bookmark_GetBookmark extends WP_UnitTestCase {
+	/**
+	 * Instance of the bookmark object.
+	 *
+	 * @var stdClass
+	 */
+	private $bookmark;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->bookmark = self::factory()->bookmark->create_and_get();
```

Comment:
Yup, must have forgotten to delete the link before exiting the class. Doh. Thanks for pointing it out to me @peterwilsoncc. Much prefer creating the bookmark when entering the test class rather than during each test (more performant).

Moved back to static property in commit https://github.com/WordPress/wordpress-develop/pull/1180/commits/829df40568b7ea437d04abdce489bebc370b575b